src/utils/cam_pio_utils.F90

```diff
@@ -940,6 +940,7 @@ subroutine cam_pio_get_var_2d_r8(varname, File, field, start, kount, found)
             strt = 1     ! cnt set by cam_pio_var_info
             exists = use_scam_limits(File, strt, cnt,filedims)
          end if
+         ! Internal PIO error handling is in effect, so need to check return codes
```

Comment:
Did you mean to say that there is no need here?

```suggestion
         ! Internal PIO error handling is in effect, so no need to check return codes
```

Reply:
Oops - fixed!